test/e2e/page-objects/pages/dialog.ts

```diff
@@ -0,0 +1,47 @@
+import { Driver } from '../../webdriver/driver';
+import { RawLocator } from '../common';
+
+class Dialog {
```

Comment:
These functions are great, but this class is super general, I was assuming we'd have classes such as `Confirmation`, `TransactionConfirmation`, and `ApprovalConfirmation` so we can modularise all the selectors and helper functions.

But since they are classes, we could also use inheritance so an `ApprovalConfirmation` instance has access to all the same functions as `TransactionConfirmation` and `Confirmation` for example?